pg-core/src/java/org/pg/Connection.java

```diff
@@ -1108,26 +1111,65 @@ private void handlePortalSuspended (final PortalSuspended msg, final Result res)
     }
 
     private void handlerCall (final IFn f, final Object arg) {
```

Comment:
There is no need to check for null here because we call this method after checking a handler for null.